packages/frontend/src/state/reducers/modals/types/index.ts

```diff
@@ -0,0 +1,3 @@
+export interface ModalsState {
+    isOpen: boolean;
```

Comment:
I'd rename this to just `open`.

Reply:
fixed: 133bb15676195f8a8a2cdb417ac0dc5fc4172360